core/addRxns.m

```diff
@@ -214,7 +214,11 @@
     for i = 1:length(rxnsToAdd.mets)
         mets         = rxnsToAdd.mets{i};
         stoichCoeffs = rxnsToAdd.stoichCoeffs{i};
-        isrev        = rxnsToAdd.lb(i) < 0;
+        if isfield(rxnsToAdd,'ub')
```

Comment:
if field `ub` does not exits, it will assume ub 1000 or inf ?

in any case if not exits, ub needs to be defined here, isn't it?

Reply:
`ub` is actually defined later, and this solution here was simpler than refactoring and moving the following code _before_ the definition of reversibility.

https://github.com/SysBioChalmers/RAVEN/blob/8cff16e8824dedb41dd75ee2dc18ddd6b097d811/core/addRxns.m#L294-L315